lib/pbench/test/functional/server/test_put.py

```diff
@@ -178,7 +178,8 @@ def test_index_all(server_client: PbenchServerClient, login_user):
             metadata=[
                 "server.tarball-path",
                 "dataset.access",
-                "global.pbench.origin",
+                "server.archiveonly",
+                "server.origin",
                 "user.pbench.access",
```

Comment:
I wonder if we should also add a test with a new ( or existing) tarball that we upload with `archiveonly` option and see at the end of test_index_all that all the datasets were indexed except the one that was uploaded with `archivelonly` option `True`?

Reply:
Ha; yeah, just finished mentioning that in a reply to Webb above. You must be psychic. Yeah, I thought about this yesterday afternoon but I wanted to get the bulk of this out for review and decided not to worry about it until today.